src/Webflow.js

```diff
@@ -52,7 +52,7 @@ const responseHandler = (res) =>
     });
 
 export default class Webflow {
-  constructor({ endpoint = DEFAULT_ENDPOINT, token, version = "1.0.0" } = {}) {
+  constructor({ endpoint = DEFAULT_ENDPOINT, token, version = "1.0.0", useCors = true } = {}) {
```

Comment:
Maybe this should be a generic `options = {}` overload that the caller can use to provide anything to fetch 🤔 It might future proof it for other factors not considered.

Reply:
Hey @johnagan! Good idea. I've made the change but called it `fetchOpts` to be a bit more specific.